wallet/src/ui/app/components/address-input/index.tsx

```diff
@@ -36,19 +41,48 @@ function AddressInput<FormValues>({
         () => SUI_ADDRESS_VALIDATION.cast(value),
         [value]
     );
+
+    const clearAddress = useCallback(() => {
+        setFieldValue('to', '');
+    }, [setFieldValue]);
     return (
-        <input
-            type="text"
-            {...{
-                disabled,
-                placeholder,
-                className,
-                onBlur,
-                value: formattedValue,
-                name,
-                onChange: handleOnChange,
-            }}
-        />
+        <div
+            className={cl(
+                st.group,
+                dirty && formattedValue !== '' && !isValid ? st.invalidAddr : ''
+            )}
+        >
+            <div className={st.textarea}>
+                <TextareaAutosize
+                    maxRows={2}
+                    minRows={1}
+                    {...{
```

Comment:
I know this is copied over from the previous version of this with an input, but there's not really a reason to do this spread vs define all properties normally.